subprojects/degrees-and-programs/source/static/app/assemble/README.md

```diff
@@ -0,0 +1,3 @@
+# brei-assemble-structure
+The BREI Assemble.io scructure
+ 
```

Comment:
Is this the same as this?

- https://github.com/BarkleyREI/generator-brei-app

Or is this the config files for that tool? I'm trying to understand the relationship between all the parts here.